### PR TITLE
Bump 1.34.1

### PR DESCRIPTION
**Pull request by Kaustubh-pande**
as per title

cc @dsimansk @pierDipi

### CODE REVIEW

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1316,11 +1316,11 @@ spec:
         - knativeeventings.operator.knative.dev
   relatedImages:
     - name: "knative-operator"
-      image: "registry.ci.openshift.org/knative/release-1.34.0:serverless-openshift-knative-operator"
+      image: "registry.ci.openshift.org/knative/main:serverless-openshift-knative-operator"
```
Comment (Kaustubh-pande) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:1319]: I think this should be release-1.34.1 but not sure why it take main by default @pierDipi @dsimansk is something i missed?

Comment (dsimansk) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:1319]: There's a env var to set during generator cmd. Just a sec... :)

Comment (dsimansk) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:1319]: https://github.com/openshift-knative/serverless-operator/blob/main/Makefile#L350-L353

The following should do the trick. 
```
CURRENT_VERSION_IMAGES=release-1.34.1 make generated-files
```

Comment (pierDipi) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:1319]: ^^ @Kaustubh-pande

**Comment by dsimansk** [comment]
@mgencur could you please take a look, if there's a way to fix upgrade tests? Or we need to have images tagged first as `release-1.34.1`?

**Comment by mgencur** [comment]
Hey. For 1.33.2 we were just removing the test from execution: https://github.com/openshift-knative/serverless-operator/commit/b4282fe789b001e0d71a4305d3037a4983a4d4f9

**Comment by skonto** [comment]
Yeah that test should be removed unless images make a difference from one S-O version to the other (not using a fixed tag).

**Comment by dsimansk** [comment]
Lint error is fine to proceed with, it's unhappy with unused functions.

**Comment by dsimansk** [comment]
@mgencur FYI the tests "stabilized" without the triggerv2 downgrade part.

**Pull request merged**